app/controllers/mock.php

```diff
@@ -605,6 +607,32 @@
             ]);
     });
 
+App::patch('/v1/mock/functions-v2')
+    ->desc('OAuth2 Failure')
```

Comment:
can you please update the descriptions, and also add the `api` & `functions` group to the `->groups()` function ?